python/ray/_private/worker.py

```diff
@@ -1641,14 +1641,29 @@ def color_for(data: Dict[str, str], line: str) -> str:
             data.get("pid") == "raylet"
             and ray_constants.LOG_PREFIX_INFO_MESSAGE not in line
         ):
-            return colorama.Fore.YELLOW
+            return colorama.Fore.RED
```

Comment:
Can you keep the same color for this? What was the motivation for the color change?

Reply:
Yes I reverted that. I had thought to mention it -- just seems like if we are going to use the color red, we should use it for errors -- but yellow works fine too.